README.md

```diff
@@ -148,3 +148,18 @@ Check out all the [published connectors](https://hub.meltano.com/singer/maintain
 | [target-yaml](https://github.com/MeltanoLabs/target-yaml)                   | Prerelease (Beta)                | 🔭 Looking For Maintainers! 🔭                    | [Link](https://hub.meltano.com/targets/yaml--meltanolabs)          |
 | [target-snowflake](https://github.com/MeltanoLabs/target-snowflake)                   | Prerelease (Beta)                | [Meltano](https://github.com/meltano) - Maintainer                    | - |
 | [target-jsonl-blob](https://github.com/MeltanoLabs/target-jsonl-blob)                   | Prerelease (Beta)                | [Edgar Ramírez](https://github.com/edgarrmondragon) - Maintainer                    | - |
+
+## Common CI Workflows
+
+MeltanoLabs taps and targets can share workflows via the GitHub [Required Workflows](https://github.blog/2023-01-10-introducing-required-workflows-and-configuration-variables-to-github-actions/#:~:text=Required%20workflows%20allows%20DevOps%20teams,impossible%20task%20in%20large%20organizations.) feature.
+
+Repositories are "opted in" to shared workflows via the [Add Required Workflows](https://github.com/organizations/MeltanoLabs/settings/actions/required_workflows/new) GitHub web UI.
+
+When eligible for this behavior, and when the repo is opted-in by a MeltanoLabs administrator, common CI workflows may include (but are not necessarily limited to):
+
+1. Automated CI testing on currently-supported Python versions.
+1. Automated verification of Semantic PR logic.
+1. Release management using dynamic versioning.
+1. Automated Release Notes drafting using Semantic PRs.
```

Comment:
> Automated Release Notes drafting using Semantic PRs

Do you intend to add a workflow for this before merging this PR?

Reply:
This is how we do it currently for Meltano and the SDK: https://github.com/meltano/sdk/blob/1ee4c4ab32bdb3b7c7e88711df2e455a2826c6ed/.github/workflows/version_bump.yml#L52-L74

Reply:
@WillDaSilva - If it is ready to go, we can certainly add it.

Reply:
@aaronsteers I just don't think it should be documented if it isn't present. I agree that we can add it later.

Reply:
On second review, I think as the workflow is defined today, it must be triggered manually.

We probably would want to make this idempotent (if not already so) and likely trigger on all pushes to `main`, so that each `main` update automatically updates/creates the release notes draft.

Reply:
Relates to:
- https://github.com/meltano/internal-tech-ops/issues/11